client/components/jetpack/card/jetpack-product-card/index.tsx

```diff
@@ -233,6 +237,9 @@ const JetpackProductCard: React.FC< OwnProps > = ( {
 				{ item.disclaimer && (
 					<span className="jetpack-product-card__disclaimer">{ item.disclaimer }</span>
 				) }
+				{ isEnabled( 'jetpack/pricing-page-product-lightbox' ) && (
+					<HintButton onClick={ onLearnMoreClick } />
```

Comment:
I was imagining the lightbox trigger as a "More about <productName>" text link inserted directly after the product description text, although I see in the PR description you describe this as a "temporary" button. So I guess we will remove the HintButton in a following PR, and replace it with a text link appended to the product description, right?

Reply:
So yes, I decided to go this way firstly to have a quick solution to be able to test a feature flag quickly and found it easier to do with a small button and `absolute` positioning not getting into messing with CSS. 
I think we could stick with it until or if business decides to go with it as it will require more time (we actually haven't estimated it) to integrate it in current cards, rather then creating a new ones. 
For example on this card there is almost no place for additional text and there is already is one `Learn more` button. So it would be quite a challenging task which will actually will be visible to users in a very short period of time (until we finish new page)
<img width="329" alt="Screenshot 2022-08-22 at 10 29 33" src="https://user-images.githubusercontent.com/60262784/185864563-b8c976b2-fefb-4d20-950a-a6bd49d7134d.png">